Example/Source/Component/Playground/ExampleView2.ui.xml

```diff
@@ -14,20 +14,18 @@
 <!--    </styles>-->
 
     <Container
-        layout:edges="super"
-        backgroundColor="#ffffff">
+        layout:edges="super">
 
         <Label
             layout:id="label"
             font=":regular@20"
             numberOfLines="0"
             textAlignment="center"
-            layout:top="[portrait] super inset(10);
-                        [landscape] super inset(0)"
+            layout:top="super inset(10)"
```

Comment:
Thanks for noticing, totally forgot about it. There's probably something wrong when running the conditions on device, because it wasn't displaying properly before removing the conditions.